valid-anagram/mand2.py

```diff
@@ -0,0 +1,33 @@
+# 문제: https://leetcode.com/problems/valid-anagram/
+
+# 풀이: s 와 t 에 입력된 알파벳의 갯수를 체크한다. ...
+# 애너그램이면 T
+def is_anagram(s, t) -> bool:
+    # 글자수가 같다는 조건이 없음.
+    if len(s) != len(t):
+        return False
+
+    word_counter = {}
+    # s 문자열 분해
+    for alpha in s:
+        # 초기화
+        if alpha not in word_counter:
+            word_counter[alpha] = 0
+        word_counter[alpha] += 1
```

Comment:
```suggestion
    word_counter = defaultdict(int)
    # s 문자열 분해
    for alpha in s:
        word_counter[alpha] += 1
```
위처럼 수정하면 #초기화 부분을 줄일 수 있는데 이건 어떨까요?

Reply:
오 감사합니다~! 이 부분을 미처 생각 못했었네요

Reply:
`defaultdict` 모듈 없이도 `get` 메서드로 기본값을 설정해줄 수도 있기는 합니다. ㅎㅎ
원하시는 방법을 사용하시면 될 것 같아요.

```python
word_counter = {}

for alpha in s:
    word_counter[alpha] = word_counter.get(alpha, 0) + 1
```

Reply:
다른 기능도 알려주셔서 감사합니다😊👍